src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -430,7 +431,10 @@ public string RoleClaimType
         /// <remarks>
         /// If set, this delegate will be called to validate the signature of the token, instead of default processing.
         /// </remarks>
-        public SignatureValidator SignatureValidator { get; set; }
+        public SignatureValidatorDelegate SignatureValidator {
+            get { return _signatureValidator; }
+            set { _signatureValidator = value; }
```

Comment:
We don't want users to set this to null, but for now let's leave it as is.

We know we have some work to do to figure this out, but the PR has enough to get us to an EndToEnd.

Reply:
Created issue #2760 to keep track of this.